test/functional/data/invalid_txs.py

```diff
@@ -245,6 +249,20 @@ def get_tx(self):
             output_script=lotsa_checksigs,
             amount=1)
 
+
+class TooManySigopsPerTransaction(BadTxTemplate):
+    reject_reason = "bad-txns-too-many-sigops"
+    expect_disconnect = False
+    valid_in_block = True
+
+    def get_tx(self):
+        lotsa_checksigs = CScript([OP_CHECKSIG] * (MAX_STANDARD_TX_SIGOPS + 1))
```

Comment:
Took me a minute to realize this is only the 2nd(?) case where it's valid in block but non-standard-even-with-acceotnonstdtxn set.

Also meta: would be nice if there was a border testing capability, where you could pair it with a tx that would be successful and obviously just-at the limit.